sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/state/RocksDBStateStoreCheckpointFormatV2Suite.scala

```diff
@@ -203,14 +210,299 @@ class RocksDBStateStoreCheckpointFormatV2Suite extends StreamTest
       withSQLConf(
         (SQLConf.STATE_STORE_PROVIDER_CLASS.key -> providerClassName),
         (SQLConf.STATE_STORE_CHECKPOINT_FORMAT_VERSION.key -> "2"),
-        (SQLConf.SHUFFLE_PARTITIONS.key, "2")) {
+        (SQLConf.SHUFFLE_PARTITIONS.key, "2"),
+        (SQLConf.STATE_STORE_MIN_DELTAS_FOR_SNAPSHOT.key, "2")
```

Comment:
can you add a comment on why this is necessary?

Reply:
added